rules/pcm/apple_framework_pcm.bzl

```diff
@@ -0,0 +1,64 @@
+load("@build_bazel_rules_swift//swift/internal:swift_common.bzl", "swift_common")
```

Comment:
@dierksen I've taken a look through the code last night and this morning to get a better grip on it - I've got some good news!

I found all of these APIs to be [officially supported public APIs](https://github.com/bazelbuild/rules_swift/issues/891#issue-1357849822) because of this - we can leverage the and don't need to tie our development of PCM compilation to rules_swift in any way